# Defining the default vision parameters

field_color_detector_path: 'basler_f031.pickle'

field_boundary_detector_search_method: 'reversed'  # iteration, reversed, binary or dynamic
field_boundary_detector_vertical_steps: 90
field_boundary_detector_horizontal_steps: 90
field_boundary_detector_roi_height: 1
field_boundary_detector_roi_width: 1
field_boundary_detector_roi_increase: 0.1  # increase of kernel radius per pixel
field_boundary_detector_green_threshold: 40
field_boundary_detector_precision_pix: 5
field_boundary_detector_min_precision_pix: 3
field_boundary_detector_head_tilt_threshold: 20

###############
## Autolabel ##
###############

dataset_roots: [
  "/srv/ssd/deep_field/data/group1/607/",
  ]

field_boundary_mask: "convex"
